docs/about_sat/introduction.md

```diff
@@ -95,48 +100,40 @@ several steps that need to be executed from a management NCN.
 
 #### Man Pages - Interactive and Non-interactive Modes
 
-To view a `sat` man page from a Kubernetes manager node, use `sat-man` on the manager node as shown in the following
-example.
+To view a `sat` man page from a Kubernetes manager node, use `sat-man` on the manager node.
+
+(`ncn-m001#`) Here is an example:
 
 ```screen
-ncn-m001# sat-man status
+sat-man status
 ```
 
 A man page describing the SAT container environment is available on the Kubernetes manager nodes, which can be viewed
 either with `man sat` or man `sat-podman` from the manager node.
 
+(`ncn-m001#`) Here are examples:
+
 ```screen
-ncn-m001# man sat
+man sat
 ```
 
 ```screen
-ncn-m001# man sat-podman
+man sat-podman
 ```
 
 ## Command Prompt Conventions in SAT
 
-The host name in a command prompt indicates where the command must be run. The account that must run the command is
-also indicated in the prompt.
+The host name in a command prompt indicates where the command must be run. The
+user account that must run the command is also indicated in the prompt.
 
-- The `root` or super-user account always has the `#` character at the end of the prompt and has the host name of the
-  host in the prompt.
-- Any non-`root` account is indicated with account@hostname>. A user account that is neither `root` nor `crayadm` is
-  referred to as `user`.
-- The command prompt inside the SAT container environment is indicated with the string as follows. It also has the "#"
-  character at the end of the prompt.
+- The `root` or super-user account always has host name in the prompt and the
+  `#` character at the end of the prompt.
+- Any non-root account is indicated with `account@hostname`. A non-privileged
+  account is referred to as user.
+- The command prompt inside the SAT container environment is indicated with the
+  string as follows. It also has the `#` character at the end of the prompt.
 
 | Command Prompt | Meaning |
 |----------------|---------|
-| `ncn-m001#` | Run on one of the Kubernetes Manager servers. (**Non-interactive**) |
+| `ncn-m001#` | Run the command on one of the Kubernetes Manager servers. (**Non-interactive**) |
```

Comment:
This is an aside, but the correct current Kubernetes terminology would be "Kubernetes Control Plane servers". I think we made the change to "managers" instead of "masters" back before Kubernetes had decided how to replace the term "master".

Source: https://kubernetes.io/docs/concepts/overview/components/

Further discussion can be found in various Kubernetes GitHub issues if you are brave enough to venture into those vitriolic discussions.

Reply:
I'll update the Kubernetes wording throughout in the next commit.